src/hash/rpo/mod.rs

```diff
@@ -145,6 +138,8 @@ impl Hasher for Rpo256 {
         // we don't need to apply any extra padding because we injected total number of elements
         // in the input list into the capacity portion of the state during initialization.
         if i > 0 {
+            state[RATE_RANGE.start + i..RATE_RANGE.start + RATE_WIDTH - 1].fill(ZERO);
```

Comment:
Why can't we use `RATE_RANGE.start + i..RATE_RANGE.end` here?

Reply:
True, fixed!